### DISCUSSION TITLE
Cheats compatible with other regions

### DISCUSSION THREAD

**Discussion by aldostools** [bucanero/ArtemisPS3#17] [General]
**Prince Of Persia BLES00389 01.01.ncl** is compatible also with **BLUS30214**. 

The cheats in this file probably work with other regions too: BLAS50085/BLKS20100/BLJM60107/NPUB30424/BLES00390/NPEL00079/BLAS50131

It's common that publishers release the same EBOOT for different regions. 
So the same concept should apply for many other games.

**Comment by bucanero** [comment]
you're right, some times codes are valid across regions.

I can add one that I recently tested to that list:
- `Heavenly Sword BCES00078 01.10.ncl` is also valid for **BCUS98132** (v1.10)

Some posted at PSX-Place:
- `Ratchet And Clank 1 BCES01503 01.00.ncl` is valid for **NPUA80643 NPEA00385** (v1.00)
-  `Ratchet And Clank 3 Up Your Arsenal BCES01503 01.00.ncl` is valid for **NPEA00387 NPUA80645** (v1.00)
